### PR TITLE
USHIFT-4916: `microshift healthcheck` - wait for `microshift.service`

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: pkg/healthcheck/greenboot.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
+package healthcheck
```
Comment (pacevedom) [inline pkg/healthcheck/greenboot.go:1]: What do you think about dropping this file and replace it with flags/options to the command instead? You would decouple the healthcheck from greenboot/grub conf reading and make things easier to reason about. Also you would be able to run the command standalone.

Comment (pmtk) [reply pkg/healthcheck/greenboot.go]: That's an excellent idea! No need to forcefully integrate with greenboot, we can keep existing timeout calculation we have in `functions.sh`

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**